realm/realm-library/src/androidTest/java/io/realm/RealmQueryTest.java

```diff
@@ -1011,6 +1011,27 @@ public void testMinValueForAllNullColumns() {
         assertNull(query.minimumDate(NullTypes.FIELD_DATE_NULL));
     }
 
+    // Test min on columns with partial null rows
+    public void testMinGivesCorrectValueForAllNonNullRows() {
```

Comment:
I would remove "GivesCorrect" from these. Hopefully that should be implicit :) So `testMinForAllNonNullRows`, same for the others.